### ISSUE TITLE
Gradle 6.8.2 breaks with MissingPropertyException

### ISSUE THREAD

**Issue by malenalbc** [kezong/fat-aar-android#274]
**Describe the issue**
Updated from Gradle version 6.7.1 to 6.8.2. When building the project after downloading the new version, it failed with this error message:

````
* What went wrong:
A problem occurred configuring project 
> No such property: buildDependencies for class: org.gradle.api.internal.artifacts.PreResolvedResolvableArtifact

* Try:
Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Exception is:
org.gradle.api.ProjectConfigurationException: A problem occurred configuring project 
(...)
Caused by: groovy.lang.MissingPropertyException: No such property: buildDependencies for class: org.gradle.api.internal.artifacts.PreResolvedResolvableArtifact
	at com.kezong.fataar.VariantProcessor.processArtifacts(VariantProcessor.groovy:263)
	at com.kezong.fataar.VariantProcessor.processVariant(VariantProcessor.groovy:66)
	at com.kezong.fataar.VariantProcessor$processVariant.call(Unknown Source)
	at com.kezong.fataar.FatAarPlugin$_doAfterEvaluate_closure3.doCall(FatAarPlugin.groovy:72)
	at org.gradle.util.ClosureBackedAction.execute(ClosureBackedAction.java:71)
	at org.gradle.util.ConfigureUtil.configureTarget(ConfigureUtil.java:154)
	at org.gradle.util.ConfigureUtil.configure(ConfigureUtil.java:105)
	at org.gradle.util.ConfigureUtil$WrappedConfigureAction.execute(ConfigureUtil.java:166)
	(...)
	at com.kezong.fataar.FatAarPlugin.doAfterEvaluate(FatAarPlugin.groovy:55)
	at com.kezong.fataar.FatAarPlugin$_apply_closure1.doCall(FatAarPlugin.groovy:40)
	(...)
````

**Build Environment**
 - Operating System: MacOS BigSur 11.2
 - Gradle Plugin Version: 4.1.2
 - Gradle Version: 6.8.2
 - Fat-aar Version: 1.3.3

**Comment by kezong** [comment]
Fixed in 1.3.4

**Comment by malenalbc** [comment]
Can confirm, updated Gradle version 6.7.1 to 6.8.3 with newest FatAAR version 1.3.4 and build run correctly.
Thanks!
